app/errors.py

```diff
@@ -0,0 +1,22 @@
+class NotWearingMaskError(Exception):
+    """
+    Not wearing mask
+    """
+
+
+class VaccineError(Exception):
+    """
+    Main vaccine error check class
+    """
+
+
+class NotVaccinatedError(VaccineError):
```

Comment:
Consider providing a default error message for your exception. For example: `class NotVaccinatedError(VaccineError): def __init__(self, message='The user is not vaccinated'): super().__init__(message)`